airbyte-integrations/connectors/source-jdbc/src/test/java/io/airbyte/integrations/source/jdbc/AbstractJdbcSourceAcceptanceTest.java

```diff
@@ -85,6 +99,18 @@ public JsonNode getConfig() {
     return config;
   }
 
+  public JsonNode getConfigWithConnectionProperties(final PostgreSQLContainer<?> psqlDb, final String dbName, final String additionalParameters) {
+    return Jsons.jsonNode(ImmutableMap.builder()
+        .put(JdbcUtils.HOST_KEY, psqlDb.getHost())
```

Comment:
nitpick: prefer `HostPortResolver.resolveHost(psqlDb)` and `HostPortResolver.resolvePort(psqlDb)`. Not strictly a problem here since we're not trying to actually connect, but I'd prefer to avoid a copy-pastable error.

(`psqlDb.getHost()` works fine on CI, but seems to cause problems when running tests on laptops)

Reply:
Just as a curiosity, what is the reason `psqlDb.getHost()` causing issues with running tests on laptops versus `HostPortResolver.resolvePort(psqlDb)`?

Reply:
I forget the specifics, but it's something about how docker's host network behaves.

I want to say that on CI (linux) you can connect to the host machine from inside a container, but on macos you need to actually use the IP from the internal docker network because docker for mac doesn't really support host networking. Or something along those lines, it's been a few weeks since I looked at this :/

(iirc switching to `host.docker.internal` as the hostname might also work? but much less confident about that)